docs/modules/ROOT/pages/reference/httpcomponents.adoc

```diff
@@ -1,12 +1,14 @@
 [[overview]]
 = Apache HttpComponents Client Instrumentation
 
-IMPORTANT: This section requires usage of Apache HttpComponents Client at least in version 5.
+IMPORTANT: This section requires usage of Apache HttpComponents Client at least in version *5.3*.
```

Comment:
It's just the retry handling that requires 5.3, right? This makes it sound like 5.3 is required for the instrumentation to work at all.